src/impl/apis/securities_api.py

```diff
@@ -142,6 +143,11 @@ def translate_historical_value(security_rate: SecurityRate) -> SecurityHistoryVa
             SecurityHistoryValue: REST resource
         """
         result = SecurityHistoryValue()
-        result.value = security_rate.rate_close
+        last_fim_date = date(1999, 1, 1)
+        fim_rate = Decimal(5.94573)
```

Comment:
Both of these values are used also in src/impl/apis/portfolios_api.py#L196, could they be moved to some constants file?

Reply:
yeah good point i fix it.

Reply:
fixed.